client/src/test/java/org/threadly/litesockets/client/ws/WebSocketClientTest.java

```diff
@@ -277,7 +277,7 @@ public void onRead(Client client) {
           for(String s: request) {
             if(s.contains(HTTPConstants.HTTP_KEY_WEBSOCKET_KEY)) {
               String[] tmp = s.split(":");
-              respKey = WebSocketFrameParser.makeKeyResponse(tmp[1].trim());
+              respKey = WSUtils.makeKeyResponse(tmp[1].trim());
```

Comment:
should the `trim()` be moved to `WSUtils.makeKeyResponse`?

Reply:
like the idea.